data/json/martialarts.json

```diff
@@ -4,6 +4,7 @@
     "id": "style_none",
     "name": "No style",
     "description": "Not a martial art, this is just plain old punching and kicking.",
+	"initiate": ["You decide to not use any martial arts.", "%s enters a generic combat stance."],
```

Comment:
Need json styling ``"initiate": [ "You decide to not use any martial arts.", "%s enters a generic combat stance." ],``
Also the indention looks off, maybe tabs depending on your editor.